sources/Context.h

```diff
@@ -123,6 +123,13 @@ class Context {
    */
   void walkNodesMain();
 
+  /**
+   * @brief Filter models
```

Comment:
Filter unused dynamic automaton

Reply:
We filter models

Reply:
yes, ok then: filter partially connected dynamic models

Reply:
OK